### DISCUSSION TITLE
gdal formula needs updating

### DISCUSSION THREAD

**Discussion by GeraldCNelson** [Homebrew/discussions#2938] [Tap maintenance and Homebrew development]
The gdal formula, at (https://formulae.brew.sh/formula/gdal), says that it depends on proj@7. But that is deprecated. Presumably this should just be 'proj'. 

The gdal formula code on GitHub at https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/gdal.rb doesn't say anything about which proj to use, but the json ap code at https://formulae.brew.sh/api/formula/gdal.json does.

I'm way beyond my expertise in home brew. Is there someone who can fix this?

**Comment by carlocab** [comment]
The formula does specify which `proj` to use here: https://github.com/Homebrew/homebrew-core/blob/e2fbb32f441c5be6fe56f227f91dcb9da523f7b2/Formula/gdal.rb#L49

**Comment by GeraldCNelson** [comment]
Ah, I see it now. So it would need to be changed there as well. I filed an issue at the github homebrew-core repository.

Gerald C. Nelson
Professor Emeritus, UIUC
+1 217-390-7888 (cell)
+1 970-639-2079 (land line)
Skype: jerrynelson
http://bit.ly/1arho7d

From: Carlo Cabrera ***@***.***>
Reply-To: Homebrew/discussions ***@***.***>
Date: Wednesday, February 9, 2022 at 8:46 AM
To: Homebrew/discussions ***@***.***>
Cc: Gerald Nelson ***@***.***>, Author ***@***.***>
Subject: Re: [Homebrew/discussions] gdal formula needs updating (Discussion #2938)


The formula does specify which proj to use here: https://github.com/Homebrew/homebrew-core/blob/e2fbb32f441c5be6fe56f227f91dcb9da523f7b2/Formula/gdal.rb#L49

—
Reply to this email directly, view it on GitHub<https://github.com/Homebrew/discussions/discussions/2938#discussioncomment-2142331>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AB6UJMQW46QYFQEM4ZXOJQ3U2KD5FANCNFSM5N53UNVA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
